store/sqlite/guest_links.go

```diff
@@ -127,17 +129,62 @@ func (s Store) DeleteGuestLink(id picoshare.GuestLinkID) error {
 	return tx.Commit()
 }
 
+func (s Store) DisableGuestLink(id picoshare.GuestLinkID) error {
+	log.Printf("Disable guest link %s", id)
```

Comment:
Log messages are not capitalized. Also, for concistency, it should be "disabling" so this should look like:

```go
log.Printf("disabling guest link %s", id)
```

(ditto for the enable function)